cookbooks/bcpc-hadoop/recipes/namenode_master.rb

```diff
@@ -72,12 +72,14 @@
   not_if { node[:bcpc][:hadoop][:mounts].any? { |d| File.exists?("/disk/#{d}/dfs/nn/current/VERSION") } }
 end
 
+zks = node[:bcpc][:hadoop][:zookeeper][:servers].map{|zkh| "#{zkh}:#{node[:bcpc][:hadoop][:zookeeper][:port]}"}.join(",")
```

Comment:
I would prefer to see this in the bash code block that's using this. I think it'd still be run in the compile phase though. I'd really like to see this in converge just in case one has to use `chef-shell` to debug this, it will be much easier.